backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -168,36 +168,46 @@ def get_filters(self, filters):
         filters_map = {
             "build": [],
             "boot": [],
-            "test": []
+            "test": [],
+            "global": [],
         }
 
         for key, value in filters.items():
-            _, filter_item = key.split("_")
+            key = key.replace("filter_", "")
 
-            filter_split = filter_item.split(".")
-            if len(filter_split) == 2:
-                table, field = filter_item.split(".")
+            table = "global"
+            if "." in key:
+                table, field = key.split(".")
             else:
-                table = "build"
-                field = filter_item
+                field = key
 
-            if table == "build":
+            if field == "valid":
+                table = "build"
                 value = [x == "Success" for x in value]
             filters_map[table].append({"field": field, "value": value})
 
         return filters_map
 
-    def pass_in_filters(self, data, filters):
+    def pass_build_filter(self, data, filters):
         for currentFilter in filters:
             field = currentFilter.get("field")
             value = currentFilter.get("value")
 
-            if (
-                field == "path"
-                and value[0] not in data[field]
-            ):
+            build_field = f"build__{field}"
+
+            if (data[build_field] not in value):
                 return False
 
+        return True
+
+    def pass_in_filters(self, data, filters):
```

Comment:
nit: rename this function to be something like "pass_test_filter" (either in singular or plural, but make it consistent with "pass_build_filter"